Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -54,6 +54,7 @@ however, it has to be formatted properly to pass verification tests.
 - Fixed "STAT event with state format TOUC cannot be used with device 'Touchscreen:/Touchscreen'" when more than max supported amount of fingers, currently 10, are present on the screen at a same time (case 1395648).
 - Fixed missing tooltips in PlayerInputManagerEditor for the Player Limit and Fixed Splitscreen sizes labels ([case 1396945](https://issuetracker.unity3d.com/issues/player-input-manager-pops-up-placeholder-text-when-hovering-over-it)).
 - Fixed DualShock 4 controllers not working in some scenarios by adding support for extended mode HID reports ([case 1281633](https://issuetracker.unity3d.com/issues/input-system-dualshock4-controller-returns-random-input-values-when-connected-via-bluetooth-while-steam-is-running), case 1409867).
+- Fixed `BackgroundBehavior.IgnoreFocus` so that it keeps the input update alive if runInBackground is false ([case 1400456](https://issuetracker.unity3d.com/issues/xr-head-tracking-lost-when-lost-focus-with-action-based-trackedposedriver-on-android)).
```

Comment:
Recommend inverting to describe the wrong behavior that was exhibited and is now fixed. Something sorta lik

```
- Fixed `BackgroundBehavior.IgnoreFocus` incorrectly getting ignored when `Application.runInBackground` was false (...).
```